### PR TITLE
Javascript single test file execution

### PR DESCRIPTION
**Pull request by Sakif-Al-Faruque**
## **User description**
In selenium documentation, javascript test file execution was missing. Therefore, I added that the command to run a test file for javascript in <a href="https://www.selenium.dev/documentation/webdriver/getting_started/first_script/#running-selenium-file">Running Selenium File > JavaScript</a> section.

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Types of changes

- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

___

## **Type**
documentation, enhancement

___

## **Description**
- Added detailed instructions on executing a single JavaScript test file in the Selenium documentation, including a specific command example.
- Introduced a `package-lock.json` file to the project structure.
- Updated the Selenium WebDriver documentation to reference the new JavaScript test execution instructions.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>README.md</strong><dd><code>Instructions for Executing a Single JavaScript Test</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
examples/javascript/README.md

<li>Added instructions on how to execute a single JavaScript test file.<br> <li> Provided a command example (<code>node example_script.spec.js</code>) for running a <br>specific test script.<br>

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1663/files#diff-69c48b40810d7b88124950eaabbdc38a117985cc0c3e284881eaff0dbd852c5b">+9/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>first_script.en.md</strong><dd><code>Documentation Update for JavaScript Test Execution</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md

<li>Updated the JavaScript tab in the "Running Selenium File" section to <br>include a reference to the new instructions for executing a single <br>JavaScript test.<br>

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1663/files#diff-0757312e3d52b8201ca35b35c4bdad191db3e0978681a45c0cad1bebeb49fd3c">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Configuration changes
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>package-lock.json</strong><dd><code>Addition of package-lock.json File</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
package-lock.json

- Introduced a new `package-lock.json` file with basic structure.

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1663/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519">+6/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Comment by Sakif-Al-Faruque** [comment]
@CLAassistant  CLA has been signed from my end. please check.

**Comment by Sakif-Al-Faruque** [comment]
> [CLA assistant check](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=1663) All committers have signed the CLA.

please check the CLA contributor sign list

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (harsha509) [inline package-lock.json]: We can exclude this file, this change is not needed!

File: website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md [hunk 2]
```diff
@@ -242,7 +242,7 @@ See [Quitting Sessions]({{< ref "../drivers/#quitting-sessions" >}}).
 {{< gh-codeblock path="examples/ruby/README.md#L36" >}}
 {{< /tab >}}
 {{< tab header="JavaScript" >}}
-{{< badge-code >}}
+{{< gh-codeblock path="examples/javascript/README.md#L36" >}}
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md:245]: Can you add similar changes to other translated files too!

See https://www.selenium.dev/documentation/about/style/#translations

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @Sakif-Al-Faruque ,

Thank you for the PR.

I have requested for few changes, can you look into them!

Thanks!

**Comment by Sakif-Al-Faruque** [comment]
> Hi @Sakif-Al-Faruque ,
> 
> Thank you for the PR.
> 
> I have requested for few changes, can you look into them!
> 
> Thanks!

My pleasure to have your opinion.

File: package-lock.json (hunk 1 above)
Comment (Sakif-Al-Faruque) [reply package-lock.json]: okay

File: website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md (hunk 2 above)
Comment (Sakif-Al-Faruque) [reply website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md:245]: oh sure. I am doing it right now.

**Comment by Sakif-Al-Faruque** [comment]
Hello, @harsha509  could you please check the last commits ?!

**Review by harsha509 — APPROVED** [review]
Thank you @Sakif-Al-Faruque for the changes!

**Comment by harsha509** [comment]
Congratulations on your first contribution to Selenium Docs @Sakif-Al-Faruque  🎉 🎉 🎉 !

**Comment by Sakif-Al-Faruque** [comment]
> Congratulations on your first contribution to Selenium Docs @Sakif-Al-Faruque 🎉 🎉 🎉 !

Great! Thank you so much @harsha509. Your important recommendations help me to make it.

**Pull request merged**